<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixel Resolution Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Qylon/Resources.qrc">
    <normaloff>:/Resources/Icon.png</normaloff>:/Resources/Icon.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="MQIT::GraphicsWidget" name="graphicsView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionStatistics"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Pixel_Resolution_Calculator"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuWindow"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="settings">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>253</width>
     <height>284</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>285</height>
    </size>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>File Configuration</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>vTools Recipe Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>csv Save Path</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QToolButton" name="toolButtonRecipe">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="Qylon/Resources.qrc">
                  <normaloff>:/Resources/Icon/icons8-opened-folder-48.png</normaloff>:/Resources/Icon/icons8-opened-folder-48.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditRecipe"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QToolButton" name="toolButtonCsv">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="Qylon/Resources.qrc">
                  <normaloff>:/Resources/Icon/icons8-opened-folder-48.png</normaloff>:/Resources/Icon/icons8-opened-folder-48.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditCsv">
                <property name="text">
                 <string/>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Execution Setting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Selected Camera</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Circle Diameter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Magnification</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QToolButton" name="toolButtonRefresh">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="Qylon/Resources.qrc">
                    <normaloff>:/Resources/Icon/icons8-refresh-48.png</normaloff>:/Resources/Icon/icons8-refresh-48.png</iconset>
                  </property>
                  <property name="arrowType">
                   <enum>Qt::ArrowType::NoArrow</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBoxDiameter">
                <property name="suffix">
                 <string> mm</string>
                </property>
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <double>10.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.000100000000000</double>
                </property>
                <property name="value">
                 <double>3.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBoxMag">
                <property name="suffix">
                 <string> X</string>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.001000000000000</double>
                </property>
                <property name="value">
                 <double>0.597000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="statistics">
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>Statistics</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QToolButton" name="toolButtonSingle">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="whatsThis">
          <string>Single</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Single</string>
         </property>
         <property name="icon">
          <iconset resource="Qylon/Resources.qrc">
           <normaloff>:/Resources/Icon/icons8-camera-48.png</normaloff>:/Resources/Icon/icons8-camera-48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonContinuous">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>Live</string>
         </property>
         <property name="text">
          <string>Live</string>
         </property>
         <property name="icon">
          <iconset resource="Qylon/Resources.qrc">
           <normaloff>:/Resources/Icon/icons8-cameras-48.png</normaloff>
           <activeon>:/Resources/Icon/icons8-pause-48.png</activeon>:/Resources/Icon/icons8-cameras-48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonExport">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>Export to csv</string>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
         <property name="icon">
          <iconset resource="Qylon/Resources.qrc">
           <normaloff>:/Resources/Icon/icons8-export-csv-48.png</normaloff>:/Resources/Icon/icons8-export-csv-48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonCopy">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>Copy Data</string>
         </property>
         <property name="text">
          <string>Copy data</string>
         </property>
         <property name="icon">
          <iconset resource="Qylon/Resources.qrc">
           <normaloff>:/Resources/Icon/icons8-copy-48.png</normaloff>:/Resources/Icon/icons8-copy-48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Pixel_Resolution_Calculator">
   <property name="text">
    <string>About This</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MQIT::GraphicsWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">Modules/GraphicsWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Qylon/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
